drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/OperatorPage.java

```diff
@@ -0,0 +1,172 @@
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.guided.dtable.client.wizard.column.pages;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Consumer;
+import javax.enterprise.context.Dependent;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import org.drools.workbench.models.datamodel.rule.BaseSingleFieldConstraint;
+import org.drools.workbench.models.guided.dtable.shared.model.ConditionCol52;
+import org.drools.workbench.screens.guided.dtable.client.resources.i18n.GuidedDecisionTableErraiConstants;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common.BaseDecisionTableColumnPage;
+import org.drools.workbench.screens.guided.dtable.client.wizard.column.plugins.ConditionColumnPlugin;
+import org.drools.workbench.screens.guided.rule.client.editor.CEPOperatorsDropdown;
+import org.gwtbootstrap3.client.ui.ListBox;
+import org.kie.workbench.common.widgets.client.datamodel.AsyncPackageDataModelOracle;
+import org.uberfire.client.callbacks.Callback;
+import org.uberfire.client.mvp.UberView;
+import org.uberfire.ext.widgets.core.client.wizards.WizardPageStatusChangeEvent;
+
+import static org.drools.workbench.screens.guided.dtable.client.wizard.column.pages.common.DecisionTableColumnViewUtils.nil;
+
+@Dependent
+public class OperatorPage extends BaseDecisionTableColumnPage<ConditionColumnPlugin> {
+
+    @Inject
+    private View view;
+
+    @Inject
+    private Event<WizardPageStatusChangeEvent> changeEvent;
+
+    @Override
+    public void initialise() {
+        content.setWidget(view);
+    }
+
+    @Override
+    public String getTitle() {
+        return translate(GuidedDecisionTableErraiConstants.OperatorPage_Operator);
+    }
+
+    @Override
+    public void prepareView() {
+        view.init(this);
+    }
+
+    @Override
+    public Widget asWidget() {
+        return content;
+    }
+
+    @Override
+    public void isComplete(final Callback<Boolean> callback) {
+        boolean hasOperator = !nil(plugin().getFactField()) && !nil(editingCol().getOperator());
+
+        callback.callback(hasOperator || isConstraintValuePredicate());
+    }
+
+    public String getOperator() {
+        return editingCol().getOperator();
+    }
+
+    void setOperator(final String selectedValue) {
```

Comment:
When my IDE hints me what is the argument by signature, `selectedValue` makes me an uncertain a little bit. But again, it may be intentional and just my problem in understanding.